### DISCUSSION TITLE
HTML Report is missing

### DISCUSSION THREAD

**Discussion by RimshaT1610** [avocado-framework/avocado#5938] [Q&A] [answered]
I have just started using avocado framework, installed it , tried sample test but I am not able to get the html report

![image](https://github.com/avocado-framework/avocado/assets/133964462/38cf3651-17af-4512-b3cd-ecf266e7e0a0)
avocado version is 105.0
Installation done via pip https://avocado-framework.readthedocs.io/en/105.0/guides/user/chapters/installing.html#installing-from-pypi
python version is 3.10.12
Using host Ubuntu 22.04 - 64-bit

**Comment by RimshaT1610** [comment, marked as answer]
The HTML report needs the html plugin enabled that is an optional plugin.

When you install through pip it doesn't come , but it's best to install via source code to have the optional_plugins directory to get it installed

**Reply by richtja** [reply]
Hi @RimshaT1610 you can also install each of the optional plugins via pip separately. They are delivered as separated package. Here you can find the [HTML plugin](https://pypi.org/project/avocado-framework-plugin-result-html/).
